### DISCUSSION TITLE
Only higher than iOS 17.4 will crashed in thread "com.apple.root.user-initiated-qos".

### DISCUSSION THREAD

**Discussion by heyingjian** [pointfreeco/swift-composable-architecture#3001] [Q&A]
Only higher than iOS 17.4 will crash. Who can help me pls？Any suggestion is acceptable.

crash Thread: #21 com.apple.root.user-initiated-qos
1. libobjc.A.dylib0x1800ea71cobjc_release_x0
2. CoreFoundation0x1881e2100
3. CoreFoundation0x1881e203c
4. libobjc.A.dylib0x1800e9f84
5. libobjc.A.dylib0x1800e9e10objc_autoreleasePoolPop
6. libdispatch.dylib0x1900d2db0
7. libdispatch.dylib0x1900e4b18
8. libdispatch.dylib0x1900e509c
9. libsystem_pthread.dylib0x1e41b8ee4_pthread_wqthread

Thread: #0 com.apple.main-thread
1. libsystem_kernel.dylib0x1d0731af8mach_msg2_trap
2. libsystem_kernel.dylib0x1d0731890mach_msg2_internal
3. libsystem_kernel.dylib0x1d07317a8mach_msg_overwrite
4. libsystem_kernel.dylib0x1d07315e8mach_msg
5. CoreFoundation0x18820c01c
6. CoreFoundation0x188209f04
7. CoreFoundation0x188209968CFRunLoopRunSpecific
8. GraphicsServices0x1cc4ff4e0GSEventRunModal
9. UIKitCore0x18a67cedc
11. UIKitCore0x18a67c518UIApplicationMain
12. APP 0x1000ee908_main (main.m:52)

**Comment by mbrandonw** [comment]
There is no mention of our library in these stack traces so there isn't much we can do to help. Can you reproduce the problem in a small project and share?
